### DISCUSSION TITLE
how to add tools from other mods to whitelist?

### DISCUSSION THREAD

**Discussion by k0ko39** [tr7zw/ItemSwapper#168] [Q&A]
as is asked, i added Vanilla hammers mod and paxel mod.
when i hold hammer or paxel, i click "r" then it display my inventory.
i looked at the config and i found the whitelist option but dont know how to use.

**Comment by tr7zw** [comment]
There is no mod whitelist? You probably confused it with the server whitelist, which is where the user has accepted that the server doesn't support this mod and that they are using the mod possibly against the servers rules. 
If you want to add modded tools to the tool list, you want to make a resource pack that adds them. Documentation is here: https://github.com/tr7zw/ItemSwapper/blob/1.20/docs/5.%20List%20Modifications.md

**Reply by joeyfigaro** [reply]
@tr7zw an example resourcepack / boilerplate for accomplishing things like this would be very useful

**Reply by tr7zw** [reply]
The wiki has basic templates, and you can use the 1.20.6 pack for 1.20.4 with experimental features as example: https://github.com/tr7zw/ItemSwapper/tree/1.20.4/Shared/src/main/resources/resourcepacks/experimental_addon
